### ISSUE TITLE
Is the TOB responsible for removing inactive maintainers in the OCI Projects?

### ISSUE THREAD

**Issue by nishakm** [opencontainers/tob#109]
One of the TOB's pain points in governance is the lack guidance on how to deal with not enough maintainers reviewing changes to individual OCI Projects.

The current charter says the following on oversight of TDCs:
The TOB shall not dictate or interfere with the day-to-day work of individual OCI Projects or their decisions.

This means that the TOB cannot effectively resolve issues in the TDC that stem from inactive maintainers. What is the TOB's role in this regard?

**Comment by dlorenc** [comment]
I think this section touches on that topic: https://github.com/opencontainers/tob/blob/main/CHARTER.md#6-technical-oversight-board-tob

> The TOB is responsible for managing conflicts, violations of procedures or guidelines and any cross-project or high-level issues that cannot be resolved in the TDC for OCI Projects.

Enough inactive maintainers could result in the inability of the active maintainers to remedy the situation.

**Comment by nishakm** [comment]
Yes, I think those two statements are contradictory :). The TOB at this time cannot resolve conflicts without interfering in some way with the goings on of the individual projects. That includes the runc project.

**Comment by dlorenc** [comment]
This was discussed about a year ago in the context of this one: https://github.com/opencontainers/tob/issues/95

I'm not speaking authoritatively, but my interpretation is that things that cannot be resolved in the TDC don't count as "day to day work" and justify the TOB acting.
